AdobeStockAdminUi/etc/di.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+  * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */-->
+
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <preference for="Magento\AdobeImsApi\Api\Data\ConfigInterface" type="Magento\AdobeIms\Model\Config"/>
```

Comment:
this preference is declared in `AdobeIms` module, it should not be declared in `AdobeStockAdminUi`